### DISCUSSION TITLE
Foxglove with WSL Ubuntu

### DISCUSSION THREAD

**Discussion by saschaflo** [foxglove/community#1012] [Q&A]
Hi all,

I installed Foxglove in WSL Ubuntu 20.04. But when I try to launch it, I receive the error message:

```
initializing Sentry in main
Loaded language from settings: undefined
Set language: en
Foxglove Studio 2.8.0
Could not set app as handler for foxglove://
Segmentation fault
```

There is a foxglove-studio.desktop file in /usr/share/applications and I have added the associations in .config/mimeapps.list:
```
[Added Associations]
application/octet-stream=foxglove-studio.desktop;
application/zip=foxglove-studio.desktop;
x-scheme-handler/foxglove=foxglove-studio.desktop;
```
But still the same error message. Would be very happy if someone could help out! 
Thanks a lot!

**Comment by achim-k** [comment]
Hard to tell from the logs what's going wrong here. Can you successfully run other WSL GUI apps? Maybe try installing VSCode in WSL and test if that works.
Do you have to use the Linux version or can you instead use the Windows version?
